test/hotspot/jtreg/gtest/LockStackGtests.java

```diff
@@ -23,10 +23,10 @@
  */
 
 /* @test
- * @summary Run LockStack gtests with LockingMode=2
```

Comment:
All gtests are executed with default vm flags in GTestWrapper.java so this while test should be just removed.